packages/design-system/src/components/mediaInput/index.js

```diff
@@ -39,6 +39,8 @@ import { PLACEMENT, Popup } from '../popup';
 import { ReactComponent as Landscape } from './landscape.svg';
 import { MEDIA_VARIANTS } from './constants';
 
+export { MEDIA_VARIANTS };
```

Comment:
Why is this re-exported here? Shouldn't other code just import from `./constants` as well?

Reply:
> Why is this re-exported here

I was not exported before, but directly access. This const, needs to be available globally. It can then be exposed in 
`export { MediaInput, MEDIA_VARIANTS } from './mediaInput';` in components/index.js